### DISCUSSION TITLE
Expose KubeDB Managed MySQL Server Using Nginx-Ingress Controller

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#65] [Blog]
# Expose KubeDB Managed MySQL Server Using Nginx-Ingress Controller

AppsCode's latest news and announcements, tips and tricks for our cloud-native ecosystem and products such as KubeDB, Stash, Voyager, ByteBuilders, KubeVault, and many more.

https://appscode.com/blog/post/expose-mysql-using-nginx-ingress/

**Comment by 7thang1** [comment]
Hi, I have a question, if I deploy it to DBaaS, can I expose multiple db to internet for client? I conflict of expose port for the service. At this line "--set tcp.3306="demo/mysql-quickstart:3306", I must use same port or I can change the port. T4R, I new to k8s ^^.
